src/main/clm_varpar.F90

```diff
@@ -56,6 +56,7 @@ module clm_varpar
 
   integer, public :: maxpatch_pft     ! obsolete: max number of plant functional types in naturally vegetated landunit (namelist setting)
   integer, public :: maxsoil_patches  ! # of pfts + cfts + bare ground; replaces maxpatch_pft, which is obsolete
+  integer, public :: n_dom_soil_patches  ! # of dominant soil patches; determines the number of active soil patches; default = 78 (= mxpft) set by namelist scripts
```

Comment:
@slevisconsulting sorry to not have thought of this sooner. But, I think a better name for n_dom_soil_patches would be n_dom_veg_patches (or even n_dom_veg?), that meshes more closely to other things we defined like maxveg, and talking about "vegetated patches". We don't really talk about "soil patches"

Reply:
@ekluzek , I'm open to changing the name; however, I picked n_dom_soil_patches because the algorithm selects among all the soil patches (from the istsoil and istcrop landunits) including the bare ground when determining the dominant. Do you still think I should change?

Reply:
Hmmm. So it includes bare-soil, so veg doesn't seem the right word to describe it. That's a good point. I'd like to run it by some folks and see what we can come up with. So I'll get back to you on this. Thanks for the explanation that's helpful.

Reply:
@ekluzek , should I add the default value to this declaration? I.e.

`integer, public :: n_dom_soil_patches = 0`

And should I leave this declaration here in clm_varpar or move to clm_varctl?

Reply:
@slevisconsulting the idea of clm_varpar is it's fixed parameters, and clm_varctl are ones that can be controlled (usually by namelist). So it would probably be best to move to clm_varctl,

Reply:
Ok, done, and gave it default value = 0

Reply:
We think the name should be changed to n_dom_pfts.

Reply:
Ok, done.